sql/core/src/test/resources/sql-tests/inputs/show-tblproperties.sql

```diff
@@ -0,0 +1,26 @@
+-- create a table with properties
+CREATE TABLE tbl (a INT, b STRING, c INT) USING parquet
+TBLPROPERTIES('p1'='v1', 'p2'='v2');
+
+SHOW TBLPROPERTIES tbl;
+SHOW TBLPROPERTIES tbl("p1");
+SHOW TBLPROPERTIES tbl("p3");
+
+DROP TABLE tbl;
+
+-- create a view with properties
+CREATE VIEW view TBLPROPERTIES('p1'='v1', 'p2'='v2') AS SELECT 1 AS c1;
+
+SHOW TBLPROPERTIES view;
+SHOW TBLPROPERTIES view("p1");
+SHOW TBLPROPERTIES view("p3");
+
+DROP VIEW view;
+
+-- create a temporary view with properties
+CREATE TEMPORARY VIEW tv TBLPROPERTIES('p1'='v1') AS SELECT 1 AS c1;
+
+-- Properties for a temporary view should be empty
```

Comment:
BTW I'd treat this as a bug. I don't think anyone would expect `SHOW TBLPROPERTIES` to always return None for temp views.

This is an existing issue, we can deal with it later.

Reply:
@cloud-fan I will create a PR for this. This can be misleading as you pointed out.

Reply:
@cloud-fan There are few ways to address this:

1. Throw an exception if `CREATE TEMPORARY VIEW` contains properties since they are ignored anyway.
2. Throw an exception when `SHOW TBLPROPERTIES` are run on a temporary view in `ShowTablePropertiesCommand`.
3. Create `UnresolvedTableOrPermanentView` and use it for commands that don't support temporary views.

WDYT?

Reply:
I'd vote for 3. We have many commands that named as XXX TABLE but can work for views.